qa/rpc-tests/test_framework/util.py

```diff
@@ -170,18 +177,38 @@ def sync_mempools(rpc_connections, wait=1,verbose=1):
             break
         time.sleep(wait)
 
+def filterUnsupportedParams(defaults, param_keys=FILTER_PARAM_KEYS):
```

Comment:
Making "flowee" the default option is not clear since the function name does not reference flowee -- this is a generic option filtering routine, and we should expect that options in other clients will also diverge.  

And the "file" option is not related to the task of  "filtering unsupported params".  Please factor the "file" out of this routine (set it explicitly in the if statements in line 206 and 209, and remove the default filter.

Reply:
file variable inside the method is removed and explicitly set in line 206 and 209.
renamed the FLOWEE_CONF to BITCOIN_CONF to make this constant generic.
BITCOIN_CONF can be set to any conf file that the client uses  (e.g. flowee.conf,  bitprim.conf, etc...)